examples/src/bin/embassy_rmt_rx.rs

```diff
@@ -12,34 +9,59 @@
 #![feature(type_alias_impl_trait)]
 
 use embassy_executor::Spawner;
+use embassy_time::{Duration, Timer};
 use esp_backtrace as _;
 use esp_hal::{
     clock::ClockControl,
     embassy::{self},
+    gpio::{Gpio5, Output, PushPull},
     peripherals::Peripherals,
     prelude::*,
     rmt::{asynch::RxChannelAsync, PulseCode, RxChannelConfig, RxChannelCreator},
-    timer::TimerGroup,
     Rmt,
     IO,
 };
 use esp_println::{print, println};
 
 const WIDTH: usize = 80;
 
+#[cfg(debug_assertions)]
+compile_error!("Run this example in release mode");
+
+#[embassy_executor::task]
+async fn signal_task(mut pin: Gpio5<Output<PushPull>>) {
+    loop {
+        for _ in 0..10 {
+            pin.toggle().unwrap();
+            Timer::after(Duration::from_micros(10)).await;
+        }
+        Timer::after(Duration::from_millis(1000)).await;
+    }
+}
+
 #[main]
-async fn main(_spawner: Spawner) {
+async fn main(spawner: Spawner) {
+    #[cfg(feature = "log")]
+    esp_println::logger::init_logger_from_env();
```

Comment:
What's the purpose of this line if none of the `log::*` macros are ever called?

Reply:
Probably copy-paste from the current original example here: https://github.com/esp-rs/esp-hal/blob/9a95c0aa880af7271f059797339bf890d2d59c64/esp32-hal/examples/embassy_rmt_rx.rs#L41-L42

It makes sense even w/o logging here to e.g. see the PSRAM logs but should get removed

Reply:
BTW I think it's also in the I2S examples in your branch .... should I remove them in my other PR then, too?

Reply:
I had removed them from the examples as they didn't seem to do anything. If you wouldn't mind that'd be great, I guess it's not a huge deal I was just curious why it was added in.

If we _do_ want to include any of these, I think they should at least have a comment stating _why_ they're there.